scholia/app/templates/base.html

```diff
@@ -25,12 +25,21 @@
 "#{{ panel }}-iframe", "{{ aspect }}_{{ panel }}.sparql");
 {%- endmacro %}
 
+{% block metas %}
+<meta charset="UTF-8">
+<meta name="robots" content="index, nofollow">
+
+<meta name="og:url" content="{{ 'https://scholia.toolforge.org' + request.path }}" />
+<meta name="og:site_name" content="Scholia" />
+<meta name="og:image" content="{{ url_for('static', filename='images/scholia_social_media.png') }}" />
+
+<meta name="twitter:card" content="summary" />
```

Comment:
At the end of last week, I was informed that Zotero cannot bookmark a `/work/Qid` page because it cannot extract the type(?), title, authors, etc of the work. I looked at the info, and they suggest they support for this standard: https://scholar.google.com/intl/en/scholar/inclusion.html#indexing which is also supported by Google Scholar and cannot but do us a lot of SEO help :)

Reply:
Wow was not aware of that! will take a look